### PR TITLE
Added 'A11y' class for screen readers

### PR DESCRIPTION
**Pull request by franciscobenedict**
[this task](https://uktrade.atlassian.net/browse/ED-135)

- Added the 'a11y' task for screen readers
- Ensured screen elements can be zoomed in +200% and and zoomed out -50% without breaking layout
- Applied context to links within forms for screen readers

screen shot 2016-10-10 at 15 38 59

### CODE REVIEW

File: ui/templates/form-wrapper.html [hunk 1]
```diff
@@ -30,7 +30,12 @@
 			<div class="register-btn-container">
 				<input class="register-btn" type="submit" value="{% trans "Register" %}"/>
 			</div>
-			<div class="already-registered">Already registered? <a class="teal" href="#">Sign in</a></div>
+			<div class="already-registered">Already registered? 
+				
+				<a class="teal" href="#">Sign in
```
Comment (richtier) [inline ui/templates/form-wrapper.html]: `aria-label` does the same job, but has less moving parts and it is more obvious what is occurring:

```html
<a class="teal" href="#" aria-label="Sign in to proceed to your company profile page">Sign in</a>
```
Is there an advantage of using the current aproach over `arial-label`?

Comment (richtier) [inline ui/templates/form-wrapper.html]: ignore this. IE8 and NVDA do not like aria-label

**Comment by konradko** [comment]
@franciscobenedict @richtier conflicts to resolve.

Also please squash commits with the same message and the "merge" ones.

**Comment by franciscobenedict** [comment]
No longer required.

**Pull request closed without merging**